docs/tutorials/pre_executed/map_partitions.ipynb

```diff
@@ -54,24 +54,18 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    # TODO: describe why or link to Olivia's dask client notebook
</pre><p>Oops, will fix</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/792/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>